app/src/main/java/ch/epfl/qedit/view/edit/EditOverviewFragment.java

```diff
@@ -60,17 +65,39 @@ public String getText(Question item) {
                                 return item.getTitle();
                             }
                         });
+    }
 
+    private void setListener() {
         adapter.setItemListener(
                 new ListEditView.ItemListener() {
                     @Override
                     public void onItemEvent(int position, ListEditView.EventType type) {
-                        if (type == ListEditView.EventType.RemoveRequest)
-                            adapter.removeItem(position);
+                        switch (type) {
+                            case Select:
+                                setFragment(new QuestionFragment());
+                                model.getFocusedQuestion().postValue(position);
+                                break;
+                            case RemoveRequest:
+                                adapter.removeItem(position);
+                                break;
+                            case EditRequest:
+                                setFragment(new EditQuestionFragment());
+                                break;
+                            default:
+                                break;
+                        }
                     }
                 });
     }
 
+    private void setFragment(Fragment fragment) {
+        getActivity()
```

Comment:
`RequireActivity()` would be safer I think.

Reply:
True